pkg/minikube/service/service.go

```diff
@@ -378,3 +378,27 @@ func DeleteSecret(cname string, namespace, name string) error {
 
 	return nil
 }
+
+func CheckServicePods(clientset typed_core.CoreV1Interface, svcName, namespace string) error {
+	svc, err := clientset.Services(namespace).Get(context.Background(), svcName, meta.GetOptions{})
+	if err != nil {
+		return errors.Wrap(err, "Fail to get service from kunernetes")
+	}
+	if svc.Spec.Type != core.ServiceTypeNodePort && svc.Spec.Type != core.ServiceTypeLoadBalancer {
+		return nil
+	}
+	pods, err := clientset.Pods(namespace).List(context.Background(), meta.ListOptions{
+		LabelSelector: labels.Set(svc.Spec.Selector).AsSelector().String(),
+	})
+	if err != nil {
```

Comment:
make wrap messages shorter

Reply:
"Fail to get pods associated with this service from kunernetes"
-->
"List Pods"